canvas_sdk/v1/data/coverage.py

```diff
@@ -46,7 +46,7 @@ class CoverageRelationshipCode(models.TextChoices):
     """CoverageRelationshipCode."""
 
     SELF = "18", "Self"
-    SPOUSE = "01" "Spouse"
+    SPOUSE = "01Spouse"
```

Comment:
I think this should be:
```suggestion
    SPOUSE = "01", "Spouse"
```

I'm guessing the linter fixed the typo incorrectly.